### ISSUE TITLE
Question regarding inverted labels

### ISSUE THREAD

**Issue by cortexml** [ssamot/DANN-keras#4]
Hi,

Thanks so much for creating this notebook, it's a very nice simple example of domain adaptation =) 

I just have a quick question regarding this line:
`stats2 = domain_classification_model.train_on_batch(X_adv, [y_adversarial_2])`

Why do you train the domain model on X0 (source) and X1 (target) and use 

`y_adversarial_2 = to_categorical(np.array(([0] * batch_size + [1] * batch_size)))`

I'm trying to wrap my head around why it's trained on inverted labels, would really appreciate your thoughts =) Thanks so much for your help!

**Comment by rebeen** [comment]
Hello, @cortexml  have you found an answer to your question ??

**Comment by cortexml** [comment]
Hi @rebeen I didn't figure this out unfortunately. Please share if you make progress! =)
